server/views/pages/checkTransfer.njk

```diff
@@ -1,75 +1,97 @@
 {% extends "../partials/layout.njk" %}
-{% set pageTitle = "Check transfer" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% set pageTitle = "Confirm prisoner's arrival" %}
```

Comment:
Not 100% sure whether page title should be 'Confirm prisoner's arrival' or 'This person is being transferred from another establishment' 
Former was shorter so used that.